src/util/message.ts

```diff
@@ -0,0 +1,98 @@
+import { decodeJWT } from '@tonomy/did-jwt';
+import { issue, OutputType } from '@tonomy/antelope-ssi-toolkit';
+import { Issuer, verifyCredential, W3CCredential } from '@tonomy/did-jwt-vc';
+
+// import { Resolver } from '@tonomy/did-resolver';
+import { getSettings } from '../settings';
+
+import { JWTDecoded } from '@tonomy/did-jwt/lib/JWT';
+import crossFetch from 'cross-fetch';
+import { getResolver } from '@tonomy/antelope-did-resolver';
+import { Resolver } from '@tonomy/did-resolver';
+// import { resolve } from './did-jwk';
+// import { getSettings } from '../settings';
+
+export class Message {
+    private decodedJwt: JWTDecoded;
+
+    /**
+     * creates a signed message and return message object
+     * @param message the messageResolver with the signer and the did
+     * @param recipient the recipient id
+     * @returns a message objects
+     */
+    static async sign(message: object, issuer: Issuer, recipient?: string): Promise<Message> {
+        const vc: W3CCredential = {
+            '@context': ['https://www.w3.org/2018/credentials/v1'],
+            // should this be tonomy foundation ?
+            id: 'https://example.com/id/1234324',
+            type: ['VerifiableCredential'],
+            issuer: {
+                id: issuer.did,
+            },
+            issuanceDate: new Date().toISOString(),
+            credentialSubject: {
+                message,
+            },
+        };
+
+        // add recipient to vc if given
+        if (recipient) vc.credentialSubject.id = recipient;
+
+        const result = await issue(vc, {
+            issuer: issuer,
+            outputType: OutputType.JWT,
+        });
+
+        return new Message(result);
```

Comment:
1. debug ideas:
- // remove Antelope SSI tookit, and use the did-jwt-vc library for issue()
- verify(result) using did-jwt-vc here

2. write a unit test in Antelope SSI Toolkit for the antelope did resolver and compare the results of resolve vs the mock resolver

3. try use the mock verify function with the mock resolver from Antelope SSI Toolkit

4. replicate a Tonomy ID account on jungle testnet, and write a test in Antelope SSI Toolkit that uses the Message class to issue and verify a VC

try in debug mode? could be black hole...